pdr-api/handlers/parks/_identifier/name/PUT/index.js

```diff
@@ -1,5 +1,7 @@
 const AWS = require("aws-sdk");
-const { TABLE_NAME, dynamodb, getOne, getSitesForProtectedArea, setSiteStatus, ESTABLISHED_STATE, HISTORICAL_STATE, REPEALED_STATE } = require('/opt/dynamodb');
+const { TABLE_NAME, dynamodb, getOne, setSiteStatus, ESTABLISHED_STATE,
+  HISTORICAL_STATE, REPEALED_STATE } = require('/opt/dynamodb');
```

Comment:
Alphabetical and vertically aligned.

Reply:
resolved in [5a73970](https://github.com/bcgov/parks-data-register/pull/434/commits/5a73970d273bbc2a88a8d3896a8446a0d62c86e1)